core/src/main/java/io/questdb/cairo/GeoHashes.java

```diff
@@ -217,7 +217,45 @@ public static void fromStringToBits(final CharSequenceHashSet prefixes, int colu
             }
         }
     }
+*/
+    public static void fromStringToBits(final CharSequenceHashSet prefixes, int columnType, final LongList prefixesBits) {
```

Comment:
Is this method used anywhere other when tests? You can perhaps move it to the test code